zerver/middleware.py

```diff
@@ -91,9 +91,10 @@ def is_slow_query(time_delta: float, path: Text) -> bool:
         return time_delta >= 10
     return True
 
-def write_log_line(log_data, path, method, remote_ip, email, client_name,
-                   status_code=200, error_content=None, error_content_iter=None):
-    # type: (MutableMapping[str, Any], Text, str, str, Text, Text, int, Optional[AnyStr], Optional[Iterable[AnyStr]]) -> None
+def write_log_line(log_data: MutableMapping[str, Any], path: Text, method: str, remote_ip: str, email: Text,
+                   client_name: Text, status_code: int=200, error_content: Optional[AnyStr]=None,
+                   error_content_iter: Optional[Iterable[AnyStr]]=None
```

Comment:
In Python, our style is to end a list with a comma when there's going to be a newline before the closing bracket -- and similarly for a dict, a parameter list like this one, and anything else delimited by commas. This makes the last item just like the other items, and in particular it means that if you add one more item at the end you don't need to go back and change this line.

(Sadly, in JavaScript this isn't allowed. But in Python it is.)

So all of these final items where the close-paren comes on the next line should get a trailing comma.